### ISSUE TITLE
[SUGGESTION]{+pic} Move "Open files in" to/under right panel (Navigation panel)

### ISSUE THREAD

**Issue by fafayqa** [Polymaker/ldd-brick-editor#20]
Another thing that seems not right visually speaking is placement of the "Open files in" segment - having it the way it is, that is in top menu line is quite non-standard for that kind of GUI object (actually I saw something like that before but it still did ring wrong to me).

So I would suggest you moving it to the actual right panel (which I hope you will move to be on the left side sometime later as it has higher priority than the left panel for individual brick values) right under the top as the 1st value/option, so it would look something like this here:
![Image 002](https://github.com/Polymaker/ldd-brick-editor/assets/26612946/3c0b8152-a39e-435a-a46a-59fe68d3e39e)

+ as I am looking at the picture now I would even suggest to changing the word "NAVIGATION" to more accurate word "PROJECT".
+ and from this point of view further on I would even suggest saving the selected "Output in folder" values with the project itself beside having it still as a value in the app settings for cases when there is no project loaded, sort of default value one selected in the settings, thus each project could have this output path set individually cos there might be cases where you want only testing that is not outputing to the LDD folder and others you want to output to the LDD folder, so one do not need constantly fiddling with the output folder per project according to its needs which even more adds to the fact changing the name of the right panel to PROJECT  and having this roll menu right under it. ;-)

**Comment by Polymaker** [comment]
> * I would even suggest saving the selected "Output in folder" values with the project itself

That's a good idea. I'll add that in my feature list. I was already planning to save the elements visual states (hidden/visible) in the project.

Otherwise the UI is inspired by Visual Studio and the selected build configuration is located at the top and I really like it. I don't think the option belongs in the same panel as the project tree.

**Comment by fafayqa** [comment]
No, I am not saying it has to be in/under project tree - project tree is just one element under Navigation panel as is now.

So once again, exactly as you see in the picture (there it is not under project tree): maybe you just mixed my thoughts (or I did mixed them?) - "Open in folder" and "Project Tree" are two objects under the same panel called at the moment "Navigation", on which I was suggesting renaming it to "PROJECT" as that would be much more appropriate once you move the "Open in folder" under it (but it's up to you, so of course, no problem if it would still be called "Navigation"), even more so if you'd implement option to save the actual value for the "Open in folder" to be saved with/per the project itself.

Hope I did explain myself better now. :-)
